Remotion/ObjectBinding/Web/UI/Controls/BocTextValueImplementation/Rendering/BocTextValueRendererBase.cs

```diff
@@ -65,6 +65,12 @@ protected ILabelReferenceRenderer LabelReferenceRenderer
       get { return _labelReferenceRenderer; }
     }
 
+    /// <inheritdoc />
+    protected override bool EstablishesThemingContext
```

Comment:
Hmm... how about `UseThemingContext` or `IsThemed`. I'm not perfectly happy, but I try to find a good `Is` or `Has` name, `Uses` being something we already have, too, I think. Maybe.

Reply:
I chose `UseThemingContext` since I think it is more expressive than `IsThemed`